src/components/breadcrumb/breadcrumb.ts

```diff
@@ -35,7 +34,9 @@ export default class SlBreadcrumb extends LitElement {
 
     // Clone it, remove ids, and slot it
     const clone = separator.cloneNode(true) as HTMLElement;
-    [clone, ...clone.querySelectorAll('[id]')].map(el => el.removeAttribute('id'));
```

Comment:
The relevant rule here is [@typescript-eslint/no-confusing-void-expression](https://typescript-eslint.io/rules/no-confusing-void-expression/) because without the curly braces, it will return the value of `removeAttribute`. Let me know if/what you'd like changed here!

Reply:
I know this can be a bit of a foot gun, but even [the example here](https://typescript-eslint.io/rules/no-confusing-void-expression/) is much more verbose. I'm OK with style over risk on this one.

Reply:
Got it. I'll remove the rule and try to find all of the code which was changed as a result.